src/main/kotlin/no/nav/syfo/cronjob/narmesteLederRevarsel/ResendNarmesteLederVarselCronjob.kt

```diff
@@ -0,0 +1,60 @@
+package no.nav.syfo.cronjob.narmesteLederRevarsel
+
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.brev.narmesteleder.NarmesteLederVarselService
+import no.nav.syfo.client.narmesteleder.NarmesteLederClient
+import no.nav.syfo.cronjob.DialogmoteCronjob
+import no.nav.syfo.cronjob.DialogmoteCronjobResult
+import no.nav.syfo.dialogmote.database.getMotedeltakerArbeidsgiverVarsel
+import no.nav.syfo.dialogmote.domain.MotedeltakerVarselType
+import no.nav.syfo.domain.PersonIdentNumber
+import no.nav.syfo.domain.Virksomhetsnummer
+import org.slf4j.LoggerFactory
+import java.time.LocalDateTime
+
+class ResendNarmesteLederVarselCronjob(
+    private val narmesteLederClient: NarmesteLederClient,
+    private val narmesteLederVarselService: NarmesteLederVarselService,
+    private val database: DatabaseInterface,
+) : DialogmoteCronjob {
+    override val initialDelayMinutes: Long = 2
+    override val intervalDelayMinutes: Long = 9999999
+
+    override suspend fun run() {
+        val resendResult = DialogmoteCronjobResult()
+
+        resendVarsler(resendResult)
+
+        log.info(
+            "Completed resending of varsler with result: {}, {}",
+            StructuredArguments.keyValue("failed", resendResult.failed),
+            StructuredArguments.keyValue("updated", resendResult.updated),
+        )
+    }
+
+    private suspend fun resendVarsler(
+        resendResult: DialogmoteCronjobResult
+    ) {
+        val varslerToResend = database.getMotedeltakerArbeidsgiverVarsel(
+            startDateTime = LocalDateTime.now().minusHours(10), // TODO: Finn riktig start
+            endDateTime = LocalDateTime.now().minusHours(5), // TODO: Finn riktig slutt
```

Comment:
Vi må ha riktig tidspunkter her 👍

Reply:
Jeg har lagt inn datoer for å teste i dev nå, så kommer proddatoer etter hvert 👍

Reply:
Jeg prøver å finne ut av de riktige tidspunktene her